source/_integrations/ads.markdown

```diff
@@ -18,6 +18,8 @@ The ADS (automation device specification) describes a device-independent and fie
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
+- [Service](#service)
```

Comment:
This is not a Table of contents. Please remove these 2 added items.

Reply:
Ok, remove it in next commit.

Reply:
This comment isn't processed. Unresolving it.